lib/exposes.js

```diff
@@ -524,6 +524,7 @@ module.exports = {
         comfort_temperature: () => new Numeric('comfort_temperature', access.STATE_SET).withUnit('°C').withDescription('Comfort temperature').withValueMin(0).withValueMax(30),
         consumer_connected: () => new Binary('consumer_connected', access.STATE, true, false).withDescription('Indicates whether a plug is physically attached. Device does not have to pull power or even be connected electrically (state of this binary switch can be ON even if main power switch is OFF)'),
         contact: () => new Binary('contact', access.STATE, false, true).withDescription('Indicates if the contact is closed (= true) or open (= false)'),
+        garage_door_contact: () => new Binary('garage_door_contact', access.STATE, false, true).withDescription('Indicates if the garage door is closed (= true) or open (= false)'),
```

Comment:
This is not used anywhere, why add it?

Reply:
Hi prepare to improve garage door opener - this is one of more patch
ie: https://github.com/Koenkk/zigbee2mqtt/pull/11680

soon I will try to create next PR for  _TZE200_nklqjk62 which will use this instead 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/3943

all becouse of https://github.com/Koenkk/zigbee2mqtt/issues/10358#issuecomment-1046348060